csvcubed/tests/unit/readers/test_preset_column_templates.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from csvcubed.readers.preconfiguredtemplates import apply_preconfigured_values_from_template
+from csvcubed.readers.vnum import JsonSchemaVersion
+
+def test_func_accepts_dict_with_template():
+    column_config = {
+        "label" : "year",
+        "from_template" : "year"
+    }
+    apply_preconfigured_values_from_template(column_config, JsonSchemaVersion.v1_0.value)
+    assert column_config.get("column_uri_template") == "http://....../{+<column_name>}"
+
+def test_func_accepts_dict_without_template():
+    column_config = {
+        "label" : "year",
+    }
+    assert "label" in column_config
+    assert "column_uri_template" not in column_config
+    assert (apply_preconfigured_values_from_template(column_config, JsonSchemaVersion.v1_0.value)) == None
```

Comment:
I don't think we need to assert anything here.

Just do something like this:

```python
from copy import deepcopy
column_config_copy = deepcopy(column_config)
...
apply_preconfigured_values_from_template(...)
assert column_config == column_config_copy
```

This will ensure that no changes have been made to the `column_config`.